RestWrite.js

```diff
@@ -654,16 +657,29 @@ RestWrite.prototype.runDatabaseOperation = function() {
     }
   }
 
+  function sendEmailVerification () {
```

Comment:
the `sendEmailVerification` function is only used in that local scope and is defined to avoid code repetition.  I guess it just depends on personal preference whether you define this within the function or in the global scope of the file.

That's a good call, I didn't even consider a user not wanting to add email verification, I'll update that.